Build with tests failure on JDK8

**Describe the bug**
When running OpenAM build on a docker `maven:3-openjdk-8` image, it fails with the following error:

```
[INFO] -----< org.openidentityplatform.openam:openam-audit-configuration >-----
[INFO] Building OpenAM Audit Configuration 14.6.5-SNAPSHOT             [27/135]
[INFO] --------------------------------[ jar ]---------------------------------
...
[ERROR] Tests run: 12, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.205 s <<< FAILURE! - in TestSuite
[ERROR] org.forgerock.openam.audit.events.handlers.JmsAuditEventHandlerFactoryTest.canCreateAuditHandlerWithBareMinimumFields  Time elapsed: 0.031 s  <<< FAILURE!
java.lang.UnsupportedClassVersionError: org/apache/activemq/jndi/ActiveMQInitialContextFactory has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at org.forgerock.openam.audit.events.handlers.JmsAuditEventHandlerFactoryTest.canCreateAuditHandlerWithBareMinimumFields(JmsAuditEventHandlerFactoryTest.java:78)
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   JmsAuditEventHandlerFactoryTest.canCreateAuditHandlerWithBareMinimumFields:78 » UnsupportedClassVersion

```
**To Reproduce**
Steps to reproduce the behavior:
1. Enter a container instance of the `maven:3-openjdk-8` image
2. Set the following env var (Workaround for PhantomJS incompatibility with Debian Buster)
        `OPENSSL_CONF: /dev/null`
3. Run `mvn -B -Dmaven.javadoc.skip=true clean install`
4. See error

**Additional context**
This is probably because the ActiveMQ client lib switched to JDK11-only in release 5.17 (https://activemq.apache.org/activemq-5017000-release) and the OpenAM POM specifies the version as an open range: 

https://github.com/OpenIdentityPlatform/OpenAM/blob/d1b2ddd0ef07a5a02d433e855f8c931216c5bd10/pom.xml#L126

Applying an upper bound to release 5.16.x should fix the issue.

--- a/pom.xml
+++ b/pom.xml
@@ -123,7 +123,7 @@
     <json.version>20090211</json.version>
     <hikaricp.version>2.4.1</hikaricp.version>
     <h2database.version>2.1.210</h2database.version>
-    <activemq.version>[5.15.9,)</activemq.version>
+    <activemq.version>[5.15.9,5.17)</activemq.version>
     <javax.inject.version>1_2</javax.inject.version>
     <jetty.jspc.version>9.4.0.M0</jetty.jspc.version>
     <amazon.sns.version>1.10.72</amazon.sns.version>

